docs/decisions/0006-make-redux-deps-optional.rst

```diff
@@ -0,0 +1,51 @@
+Consumers of frontend-platform that don't use redux are still required to install redux dependencies
+==========================================
+
+Status
+------
+
+Review
+
+Context
+-------
+
+Some consumers of frontend-platform, such as frontend-app-learner-portal-enterprise, no longer use redux or react-redux.
+However, they are still required to install these dependencies due to the usage of react-redux in AppProvider, and the fact that
+react-redux and redux are peerDependencies.
+
+This means there is extra overhead and confusion in such projects, when they are not leveraging the `store` feature of AppProvider.
+
+It will be great to be able to not have to include redux and react-redux as dependencies in such cases.
+
+Decision
+--------
+
+A method to enable the removal of the dependencies `redux` and `react-redux` from mentioned client projects is to leverage optionalDependencies.
+
+According to npm documentation:
```

Comment:
nit: link to documentation? https://docs.npmjs.com/cli/v6/configuring-npm/package-json#optionaldependencies

(note: this link is specifically for NPM 6, but we should probably link to latest version.